tests/e2e/test_e2e_jobs.py

```diff
@@ -591,6 +591,22 @@ def test_e2e_ssh_exec_tty(helper: Helper) -> None:
     assert captured.out == ""
 
 
+@pytest.mark.e2e
+def test_e2e_ssh_exec_tty_is_by_default(helper: Helper) -> None:
```

Comment:
since exec tests are quite flaky, maybe we might want to consider removing (or at least muting) the other test: `test_e2e_ssh_exec_tty`